ImperatorToCK3/Data_Files/options_l_english.yml

```diff
@@ -7,6 +7,13 @@ l_english:
  IMPERATORDEJURE2TIP: "Base CK3 de iure on Imperator countries."
  IMPERATORDEJURE3: "No"
  IMPERATORDEJURE3TIP: "Keep vanilla CK3 de iure."
+ 
+ CONVERTCHARACTERBIRTHANDDEATHDATES: "Base for character birth date conversion"
```

Comment:
Where are other languages?

Reply:
Who cares?

Reply:
Well, quite obviously, I do. And you'll feel warm inside knowing you support russian even through autotranslate once they come knocking. :)

Reply:
I'll add other langs when I make a Python script to add Google translation for me. We talked about it on Discord some time ago.

Reply:
Added to 0.3 milestones.